mitodl/open-discussions
# Refactor widgets and style read-only widgets

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #1573 

#### What's this PR do?
Removes the parts of the frontend widget code which add and edit widgets. This will require major refactoring so it will be easier to 

#### How should this be manually tested?
View widgets in a channel page. If you throttle the network traffic you should see a loading indication while the widgets load. Widgets should look similar to they do in the Zeplin mockup here: https://app.zeplin.io/project/5b181d3bf71408ae4d406b94/screen/5ba4e6e6c699e4552e267660

You should be able to PATCH the widgets by editing the JSON blob at the `/api/v0/widget_lists/{widget_id}/` endpoint
